bigbluebutton/greenlight
# Accessibility navigate and create room

Hi, This PR will enable keyboard users to be able to navigate through rooms and also create a new room.
On current greenlight, keyboard user cannot create a new room, if you try to navigate using keyboards (tabs key), you will be looped on the first room link and its settings button. 
This PR main goal is to make keyboard user able to reach the new create room button.

## Description

To do this, we change the room element from <a> tag to <div> and we add some some action on it, so that enter key and escape key still works.
We also add some aria-label on the tags so that screen reader can read it.

## Testing Steps

To test it:
1. Go to greenlight dashboard or room
2. Try to open a create room modal by using only keyboard
3. You should be able to navigate to the new create room button using only keyboard

## Screenshots (if appropriate):

There should not be any changes in term of visual.
